src/App.js

```diff
@@ -1,10 +1,23 @@
+/* eslint-disable  */
 import './App.css';
 import Calculator from './components/Calculator';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Home from './components/Home';
+import Quote from './components/Quote';
+import SharedComponent from './components/SharedComponent';
 
 function App() {
   return (
     <div>
-      <Calculator />
+      <BrowserRouter>
+        <Routes>
+          <Route path='/' element={<SharedComponent />}>
+            <Route path='home' element={<Home />} />
```

Comment:
- [ ] Hi @John-Kibirige, I think your Home page should be the root route. Right now `http://localhost:3000/` display a blank page which does not give a good user experience. Let's fix this issue.